docs/source/reference/samplers/index.rst

```diff
@@ -11,6 +11,40 @@ The :mod:`~optuna.samplers` module defines a base class for parameter sampling a
 .. seealso::
     :ref:`user_defined_sampler` tutorial could be helpful if you want to implement your own sampler classes.
 
++----------------------------------+---------------+-------------+-------------+--------------+----------------------------------------+------------+----------------+
+|                                  | RandomSampler | GridSampler | TPESampler  | CmaEsSampler |              NSGAIISampler             | QMCSampler | BoTorchSampler |
++==================================+===============+=============+=============+==============+========================================+============+================+
+| Float parameters                 |       ✅      |     ✅      |     ✅      |      ✅      | ▲(✅ if you use non-default crossover) |     ✅     |       ✅       |
++----------------------------------+---------------+-------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Integer parameters               |       ✅      |     ✅      |     ✅      |      ✅      | ▲(✅ if you use non-default crossover) |     ✅     |       ✅       |
++----------------------------------+---------------+-------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Categorical parameters           |       ✅      |     ✅      |     ✅      |      ❌      |                   ✅                   |     ❌     |       ✅       |
++----------------------------------+---------------+-------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Multivariate optimization        |       ❌      |     ❌      |     ✅      |      ✅      |                   ❌                   |     ❌     |       ✅       |
++----------------------------------+---------------+-------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Conditional search space         |       ❌      |     ❌      |     ✅      |      ❌      |                   ❌                   |     ❌     |       ❌       |
++----------------------------------+---------------+-------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Multi-objective optimization (*) |       ▲       |     ▲       |     ✅      |      ❌      |       ✅(▲ for single-objective)       |     ▲      |       ✅       |
++----------------------------------+---------------+-------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Batch optimization               |       ❌      |     ❌      |     ✅      |      ❌      |                   ❌                   |     ❌     |       ❌       |
```

Comment:
I'd say RandomSampler and GridSampler do not have any restrictions for Batch optimization since it doesn't need to consider past evaluations, and CmaEsSampler supports Batch optimization though Optuna users may be difficult to choose appropriate number of batches.

What do you think?

```suggestion
| Batch optimization               |       ✅      |     ✅      |     ✅      |      ▲      |                   ❌                   |     ❌     |       ❌       |
```